pkg/orchestration/wiring/kubeingress/plugin.go

```diff
@@ -61,30 +60,24 @@ func WireUp(resource *orch_v1.StateResource, context *wiringplugin.WiringContext
 	deploymentName := deploymentSpec.Name
 	deploymentLabels := deploymentSpec.Spec.Template.ObjectMeta.Labels
 
-	var smithResources []wiringplugin.WiredSmithResource
-
 	// Build the Service
-	serviceResource := buildServiceResource(smith_v1.ResourceName(deploymentName), deploymentLabels, resource, context)
-	smithResources = append(smithResources, serviceResource)
+	serviceResource := buildServiceResource(smith_v1.ResourceName(deploymentName), deploymentLabels, resource.Name)
```

Comment:
`smith_v1.ResourceName(deploymentName)` is a bug. Resource name of the resource can be different from the name of the `Deployment` object itself. Not fixing here, as said in the description.